spec/models/participant_outcome_spec.rb

```diff
@@ -87,4 +95,130 @@
+        it { is_expected.to include(outcome_3.id) }
+      end
+
+      context "when the latest outcome is not passed" do
+        let!(:outcome_1) { create(:participant_outcome, :passed, :not_sent_to_qualified_teachers_api, declaration:) }
+        let!(:outcome_2) { create(:participant_outcome, :failed, :not_sent_to_qualified_teachers_api, declaration:) }
+        let!(:outcome_3) { create(:participant_outcome, :voided, :not_sent_to_qualified_teachers_api, declaration:) }
+
+        it { is_expected.not_to include(outcome_1.id, outcome_2.id, outcome_3.id) }
+      end
+    end
+
+    describe ".sent_to_qualified_teachers_api" do
+      let!(:outcome_1) { create(:participant_outcome, :sent_to_qualified_teachers_api) }
+      let!(:outcome_2) { create(:participant_outcome, :not_sent_to_qualified_teachers_api) }
+
+      subject(:result) { described_class.sent_to_qualified_teachers_api.map(&:id) }
+
+      it { is_expected.to include(outcome_1.id) }
+      it { is_expected.not_to include(outcome_2.id) }
+    end
+
+    describe ".not_sent_to_qualified_teachers_api" do
+      let!(:outcome_1) { create(:participant_outcome, :sent_to_qualified_teachers_api) }
+      let!(:outcome_2) { create(:participant_outcome, :not_sent_to_qualified_teachers_api) }
+
+      subject(:result) { described_class.not_sent_to_qualified_teachers_api.map(&:id) }
+
+      it { is_expected.to include(outcome_2.id) }
+      it { is_expected.not_to include(outcome_1.id) }
+    end
+
+    describe ".declarations_where_outcome_passed_and_sent" do
+      let(:application_1) { create(:application, :accepted) }
+      let(:declaration_date_1) { application.schedule.applies_from + 1.day }
+      let(:application_2) { create(:application, :accepted) }
+      let(:declaration_date_2) { application.schedule.applies_from + 1.day }
+      let!(:declaration_1) do
+        travel_to declaration_date_1 do
+          create(:declaration, :completed, application:, declaration_date:)
+        end
+      end
+      let!(:declaration_2) do
+        travel_to declaration_date_2 do
+          create(:declaration, :completed, application:, declaration_date:)
+        end
+      end
+
+      before do
+        create(:participant_outcome, :failed, :sent_to_qualified_teachers_api, declaration: declaration_1)
+        create(:participant_outcome, :passed, :not_sent_to_qualified_teachers_api, declaration: declaration_2)
+        create(:participant_outcome, :passed, :sent_to_qualified_teachers_api, declaration: declaration_2)
+        create(:participant_outcome, :voided, :sent_to_qualified_teachers_api, declaration: declaration_2)
+      end
+
+      subject(:result) { described_class.declarations_where_outcome_passed_and_sent }
+
+      it { is_expected.not_to include(declaration_1.id) }
+      it { is_expected.to include(declaration_2.id) }
+    end
+
+    describe ".latest_per_declaration" do
+      let(:application_1) { create(:application, :accepted) }
```

Comment:
Is it worth having a helper method in here to create outcomes, declarations and applications to reduce the amount of setup in this spec?

```
def create_outcome
  application = create(:application, :accepted)
  declaration_date = application.schedule.applies_from + 1.day
  travel_to declaration_date_2 do
    declaration = create(:declaration, :completed, application:, declaration_date:)
  end
  create(:participant_outcome, declaration:)
end
```